pkgs/by-name/ga/gambit-project/package.nix

```
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, wxGTK31
+, lib
+, withGui ? true
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gambit-project";
+  version = "16.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gambitproject";
+    repo = "gambit";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ElPzJDQ1q+i1OyliychSUA9pT6yGSwjn/sKV0JX5wrQ=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = lib.optional withGui wxGTK31;
+
+  meta = with lib; {
+    description = "An open-source collection of tools for doing computation in game theory";
+    homepage = "http://www.gambit-project.org";
+    license = licenses.gpl2Only;
+    maintainers = with maintainers; [ t4ccer ];
+    platforms = platforms.linux ++ platforms.darwin ++ platforms.windows;
```

Let's get rid of nested with:

```suggestion
  meta = {
    description = "An open-source collection of tools for doing computation in game theory";
    homepage = "http://www.gambit-project.org";
    license = lib.licenses.gpl2Only;
    maintainers = with lib.maintainers; [ t4ccer ];
    platforms = with lib.platforms; unix ++ windows;
```